lua/astrocommunity/color/ccc-nvim/ccc-nvim.lua

```diff
@@ -7,7 +7,6 @@ return {
     opts = { highlighter = { auto_enable = true } },
     config = function(_, opts)
```

Comment:
can u try to remove the whole config part?

Reply:
looks like that worked, assuming just a error for pointless config section?